apiserver/common/tools.go

```diff
@@ -256,6 +251,23 @@ func (f *ToolsFinder) findTools(args params.FindToolsParams) (coretools.List, er
 	if err != nil {
```

Comment:
Do we get a list back with a 2.8 client or does findMatchingTools return an error if the client provided `args.Series`?

Reply:
We will get a list in these cases. All this does is massage the return based on the args. if you requested by series, you get series-name agents; if by OS, then OS-name agents, no matter what was returned.